docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -0,0 +1,476 @@
+
+This feature is disabled by default. To enable it, use the `metricsSummary` [experimental feature toggle](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/feature-toggles/#experimental-feature-toggles).
+
+For more information, refer to the [documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/query-editor/traceql-search/#optional-use-aggregate-by).
+
+{{< figure src="/media/docs/tempo/metrics-summary-10-2.png" caption="Aggregate by" >}}
+
+#### Query traces more easily with the Improved TraceQL editor
+
+<!-- Fabrizio Casati -->
+
+_Generally available in all editions of Grafana_
+
+The [TraceQL query editor](https://grafana.com/docs/tempo/latest/traceql/#traceql-query-editor) has been improved to facilitate the creation of TraceQL queries. In particular, it now features improved autocompletion, syntax highlighting, and error reporting.
+
+{{< video-embed src="/media/docs/tempo/screen-recording-grafana-10.2-traceql-query-editor-improvements.mp4" >}}
+
+#### Group multiple spansets per trace
+
+<!-- Joey Tawadrous -->
+
+_Generally available in all editions of Grafana_
+
+The [TraceQL query editor](https://grafana.com/docs/tempo/<TEMPO_VERSION>/traceql/#traceql-query-editor) has been improved to facilitate the grouping of multiple spans per trace in TraceQL queries. For example, when the `by(resource.service.name)` is added to your TraceQL query, it will group the spans in each trace by `resource.service.name`.
+
+{{< figure src="/media/docs/tempo/multiple-spansets-per-trace-10-2.png" max-width="750px" caption="Multiple spansets per trace" >}}
+
+#### Create Query-type template variables for the Tempo data source
+
+<!-- Fabrizio Casati -->
+
+_Generally available in all editions of Grafana_
+
+The Tempo data source now supports query-type template variables. With this update, you can create variables for which the values are a list of attribute names or attribute values seen on spans received by Tempo.
+
+To learn more, refer to the following video demo, as well as the [Grafana Variables documentation](/docs/grafana/next/dashboards/variables/).
+
+{{< video-embed src="/media/docs/tempo/screen-recording-grafana-10.2-tempo-query-type-template-variables.mp4" >}}
+
+### SAP HANA®: Configure your data source with tenant database instance name and number
+
+<!-- Miguel Palau -->
+
+_Generally available in Grafana Enterprise and Grafana Cloud Advanced_
+
+The SAP HANA® data source now supports tenant database connections by using the database name and/or instance number, which are less likely to change than the port for your database. For more information, refer to our [SAP HANA® configuration documentation](/docs/plugins/grafana-saphana-datasource/latest/#configuration).
```

Comment:
```suggestion
The SAP HANA® data source now supports tenant database connections by using the database name and/or instance number. This is helpful because these are less likely to change than the port for your database. For more information, refer to our [SAP HANA® configuration documentation](/docs/plugins/grafana-saphana-datasource/latest/#configuration).
```